README.md

```diff
@@ -35,4 +35,37 @@ and you will get:
  'CMIP6.PMIP.MPI-M.MPI-ESM1-2-LR.lgm.r1i1p1f1.Omon.vo.gn.v20190710']
 ```
 
-Eventually I hope I can leverage this functionality to handle user requests in PRs that add wildcard instance_ids, but for now this might be helpful to manually construct lists of instance_ids to submit to a pangeo-forge feedstock.
\ No newline at end of file
+Eventually I hope I can leverage this functionality to handle user requests in PRs that add wildcard instance_ids, but for now this might be helpful to manually construct lists of instance_ids to submit to a pangeo-forge feedstock.
+
+
+## Use different ESGF project
+
+You can also use different ESGF projects and facets to search, e.g., for a search in the ESGF CORDEX datasets, use:
+```python
+parse_iids = [
+    "CORDEX.output.EUR-11.MPI-CSC.MPI-M-MPI-ESM-LR.*.r1i1p1.REMO2009.v1.mon.tas",
+    "CORDEX.output.EUR-44.MPI-CSC.MPI-M-MPI-ESM-LR.*.r1i1p1.REMO2009.v1.mon.tas"
+]
+iids = []
+```
+
+using a different url:
```

Comment:
Can you clarify if this is necessary or if this is just to show that the functionality works on all the ESGF search nodes?

Reply:
Yes, that was just to show, that you can choose a different url. CORDEX searches should actually also work for all ESGF nodes.